service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AisConsentSpiImpl.java

```diff
@@ -41,14 +41,15 @@ public AisConsentSpiImpl(AuthorisationService authorisationService,
       SpiAccountConsent spiAccountConsent,
       AspspConsentData aspspConsentData) {
 
+    // TODO potential bug - we need to check ALL IBANs here? (rat)
```

Comment:
It should be enough to just take the first iban to get a PSU. If this or any other provided Iban does not match the returned PSU, we will receive errors during SCA.

Reply:
Depending on the order in the accounts you might get FORMAT_ERROR or an error during SCA, right? This feels odd. But good enough for now.